scripts/modules/elastic_stack.py

```diff
@@ -844,6 +853,12 @@ def add_arguments(cls, parser):
             help="disable the APM service maps UI",
         )
 
+        parser.add_argument(
+            "--kibana-from-sources",
```

Comment:
wdyt about renaming the flag to something easier to recall, like `--kibana-src`?